graphql/schemabuilder/helpers.go

```diff
@@ -92,6 +93,20 @@ func (sb *schemaBuilder) constructConnType(typ reflect.Type) (graphql.Type, erro
 
 	fieldMap := make(map[string]*graphql.Field)
 
+	countType, err := sb.getType(reflect.TypeOf(int64(0)))
+	countField := &graphql.Field{
+		Resolve: func(ctx context.Context, source, args interface{}, selectionSet *graphql.SelectionSet) (interface{}, error) {
+			if value, ok := source.(Connection); ok {
+				return value.Count, nil
+			}
+			return nil, fmt.Errorf("error resolving count in connection")
+		},
+		Type:           countType,
+		ParseArguments: nilParseArguments,
+	}
+
+	fieldMap["count"] = countField
```

Comment:
what do you think about running  [`buildField`](https://sourcegraph.com/github.com/samsarahq/thunder@34f512681aef54069ee033ba45660fdb168a6d8e/-/blob/graphql/schemabuilder/reflect.go#L579:26) on these instead of building them by hand?

Reply:
I think that's only possible with count.